public/index.d.ts

```diff
@@ -2,8 +2,8 @@ import { IncomingMessage, ServerResponse } from "http";
 import { GraphQLScalarType } from "graphql";
 import { ReadStream } from "fs";
 
-import { RequestHandler } from "@types/express";
-import { DefaultContext, DefaultState, Middleware } from "@types/koa";
+import { RequestHandler } from "express";
```

Comment:
Can we `import type ...` instead of `import` ?
Asking for a friend. :)

Reply:
Added!

Thank you for reopening this PR.